mqtt-transport-binding.md

```diff
@@ -176,7 +176,8 @@ in the MQTT PUBLISH message.
 ##### 3.1.3.2 User Property Values
 
 The value for each MQTT PUBLISH User Property MUST be constructed from the
-respective CloudEvents attribute type's canonical string representation.
+respective CloudEvents attribute type's canonical string representation in the
+[JSON][json-format] string encoding.
```

Comment:
We define the canonical representations here. We've decoupled them from JSON now.

Reply:
Thanks, updated the link to the type system